_posts/en/newsletters/2023-10-18-newsletter.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 'Bitcoin Optech Newsletter #273'
+permalink: /en/newsletters/2023/10/18/
+name: 2023-10-18-newsletter
+slug: 2023-10-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Payments contingent on arbitrary computation:** Robin Linus
+  [posted][linus post] to the Bitcoin-Dev a [paper][linus paper] he's
+  written about _BitVM_, a combination of methods that allows bitcoins
+  to be paid to someone who successfully proves that an arbitrary
+  program executed successfully.  Notably, this is possible on Bitcoin
+  today---no consensus change is required.
+
+    To provide context, a well-known feature of Bitcoin is to require
+    someone to satisfy a programming expression (called a _script_) in
+    order to spend bitcoins associated with that script.  For example, a
+    script containing a public key that can only be satisfied if the
+    corresponding private key creates a signature committing to a
+    spending transaction.  Scripts must be written in Bitcoin's language
+    (called _Script_) in order to be enforced by consensus, but Script
+    is deliberately limited in its flexibility.
+
+    Linus's paper gets around some of those limits.  If Alice trusts Bob
+    to take action if a program was run incorrectly, but does not want
+    to trust him with anything else, she can pay funds to a taproot tree
+    that will allow Bob to claim the funds if he demonstrates that Alice
+    failed to run an arbitrary program correctly.  If Alice does run the
+    program correctly, then she can spend the funds even if Bob attempts
+    to stop her.
+
+    To use an arbitrary program, it must be broken down into a very
+    basic primitive (a NAND gate) and a commitment must be made for each
+    gate.  This requires the offchain exchange of a very large amount of
+    data, potentially multiple gigabytes for even a fairly basic
+    arbitrary program---but Alice and Bob only need a single onchain
+    transaction in the case that Bob agrees that Alice ran the program
+    correctly.  In the case that Bob disagrees, he should be able to
+    demonstrate Alice's failure within a relatively small number of
+    onchain transactions.  If the setup was performed in a payment
+    channel between Alice and Bob, multiple programs could be run in
+    both parallel and in sequence with no onchain footprint except
```

Comment:
Feels like to me this wants to read 
"could be run both in parallel and in sequence with no onchain"